client/src/pages/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,154 @@
+              <Box className={classes.box}>
+                <label htmlFor="first-name" className={classes.label}>
+                  FIRST NAME
+                </label>
+                <TextField type="text" id="first-name" placeholder="john" variant="outlined" style={{ width: '70%' }} />
+              </Box>
+              <Box className={classes.box}>
+                <label htmlFor="last-name" className={classes.label}>
+                  LAST NAME
+                </label>
+                <TextField type="text" id="last-name" placeholder="john" variant="outlined" style={{ width: '70%' }} />
+              </Box>
+              <Box className={classes.box}>
+                <label htmlFor="first-name" className={classes.label}>
+                  GENDER
+                </label>
+                <FormControl variant="outlined">
+                  <Select
+                    labelId="demo-simple-select-outlined-label"
+                    id="demo-simple-select-outlined"
+                    value={gender}
+                    onChange={() => {}}
+                    style={{ width: '100px' }}
+                  >
+                    <MenuItem value={'male'}>Male</MenuItem>
+                    <MenuItem value={'female'}>Female</MenuItem>
+                  </Select>
+                </FormControl>
+              </Box>
+              <Box className={classes.box}>
+                <label htmlFor="first-name" className={classes.label}>
+                  BIRTH DATE
+                </label>
+                <Box component="span" style={{ width: '70%' }}>
+                  <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                    <DatePicker
+                      views={['month']}
+                      inputVariant="outlined"
+                      value={selectedDate}
+                      onChange={() => {}}
+                      style={{ width: '100px' }}
+                    />
+                  </MuiPickersUtilsProvider>
+                  <FormControl variant="outlined">
+                    <Select
+                      labelId="demo-simple-select-outlined-label"
+                      id="demo-simple-select-outlined"
+                      style={{ width: '100px' }}
+                    >
+                      {optionSelect}
+                    </Select>
+                  </FormControl>{' '}
+                  <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                    <DatePicker
+                      views={['year']}
+                      inputVariant="outlined"
+                      value={selectedDate}
+                      onChange={() => {}}
+                      style={{ width: '100px' }}
+                    />
+                  </MuiPickersUtilsProvider>
+                </Box>
+              </Box>
+              <Box className={classes.box}>
+                <label htmlFor="phone-number" className={classes.label}>
+                  PHONE NUMBER
```

Comment:
instead of cap locking this, i would just tweak it with CSS to display in all caps. it's not that great for SEO and maintainability :)

Reply:
changed to css style  {    textTransform: 'uppercase', }